readthedocs/core/views/serve.py

```diff
@@ -215,3 +215,36 @@ def _serve_symlink_docs(request, project, privacy_level, filename=''):
 
     raise Http404(
         'File not found. Tried these files: %s' % ','.join(files_tried))
+
+
+@map_project_slug
+@map_subproject_slug
+def robots_txt(request, project, subproject, lang_slug=None, version_slug=None):
+    default_robots_fullpath = os.path.join(settings.MEDIA_ROOT, 'robots.txt')
+
+    if not version_slug:
+        version_slug = project.get_default_version()
```

Comment:
We should consider make the same decision as for custom 404 pages here (https://github.com/rtfd/readthedocs.org/issues/2551#issuecomment-453046108) about which version to use.